boards/shields/adafruit_drv2605l/adafruit_drv2605l.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2025 Jonas Berg
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&zephyr_i2c {
+	status = "okay";
+
+	adafruit_drv2605: drv2605@5a {
+		compatible = "ti,drv2605";
+		reg = <0x5a>;
+		status = "okay";
+		actuator-mode = "ERM";
+	};
+};
+
+/* Alternate node label for samples/drivers/haptics/drv2605 */
+haptic: &adafruit_drv2605 {};
```

Comment:
would be nice to use an alias in the sample, but this is of course very much out of scope for this PR